### ISSUE TITLE
Real debrid plugin problem

### ISSUE THREAD

**Issue by Dybala93** [pyload/pyload#1664]
Required account not found:
log:
INFO	ACCOUNT RealdebridCom: Relogin user `*`...
WARNING	ACCOUNT RealdebridCom: Could not login user `*` | 'password'

Restarting pyload work again...but same problem after 15-30 mins!
Thanks for help!

**Comment by vuolter** [comment]
Debug log, please.

**Comment by cedmeister** [comment]
Same here... Worked a few hours ago.

**Comment by Dybala93** [comment]
debug log:
```
INFO	Download starts: 1.rar
DEBUG	HOOK UserAgentSwitcher: Use custom user-agent string: Mozilla/5.0 (Windows NT 6.1; WOW64; rv:38.0) Gecko/20100101 Firefox/38.0
DEBUG	ACCOUNT RealdebridCom: Reached login timeout for *
INFO	ACCOUNT RealdebridCom: Relogin user `*`...
WARNING	ACCOUNT RealdebridCom: Could not login user `*` | 'password'
DEBUG	HOOK UserAgentSwitcher: Use custom user-agent string: Mozilla/5.0 (Windows NT 6.1; WOW64; rv:38.0) Gecko/20100101 Firefox/38.0
DEBUG	HOSTER RealdebridCom[217]: RealdebridCom | PROCESS URL ...1.rar | PLUGIN VERSION 0.69
WARNING	Download failed: 1.rar | Required account not found
```

And what is the function of HOOK UserAgentSwitcher?
Thanks vuolter!

**Comment by vuolter** [comment]
@Dybala93 UserAgentSwitcher simply changes the user-agent used by pyload with what you want.

**Comment by Dybala93** [comment]
Thanks for help but for me not solved.
Same problem!

**Comment by vuolter** [comment]
@Dybala93 Okay, debug log like  @cedmeister one.

**Comment by Dybala93** [comment]
maybe I replaced the wrong  file.
Now i've found and replaced the file Account.py in /pyload/data/userplugins/internal and I'm testing if now it works correctly!

I think that i replaced the wrong account.py in .qpkg/pyload/module/plugins

Sorry for my ignorance! I'm testing now!

**Comment by Dybala93** [comment]
Ok, nothing...after 15-30 mins same problem.
Sorry for all time losing for my ignorance!
In data folder I have only debug_tusfiles.net (I use rd plugin for tusfiles host) but i'm not sure that this is the debug log that you want. If not where i find that like cedmeister?
tusfiles debug (download link replaced with "link"):
```
pyLoad 0.4.9 Debug Report of TusfilesNet 0.12 

TRACEBACK:
 Traceback (most recent call last):
  File "/share/HDA_DATA/.qpkg/pyload/module/PluginThread.py", line 187, in run
    pyfile.plugin.preprocessing(self)
  File "/share/HDA_DATA/.qpkg/pyload/data/userplugins/internal/Hoster.py", line 184, in preprocessing
    return self.process(self.pyfile)
  File "/share/HDA_DATA/.qpkg/pyload/data/userplugins/internal/SimpleHoster.py", line 281, in process
    self.preload()
  File "/share/HDA_DATA/.qpkg/pyload/data/userplugins/internal/SimpleHoster.py", line 255, in preload
    decode=self.TEXT_ENCODING)
  File "/share/HDA_DATA/.qpkg/pyload/data/userplugins/internal/Plugin.py", line 332, in load
    res = req.load(url, get, post, ref, cookies, just_header, multipart, decode is True)  #@TODO: Fix network multipart in 0.4.10
  File "/share/HDA_DATA/.qpkg/pyload/module/network/Browser.py", line 97, in load
    return self.http.load(*args, **kwargs)
  File "/share/HDA_DATA/.qpkg/pyload/module/network/HTTPRequest.py", line 202, in load
    self.c.perform()
error: (23, 'Failed writing body (0 != 16234)')
 

FRAMESTACK:

Frame preprocessing in /share/HDA_DATA/.qpkg/pyload/data/userplugins/internal/Hoster.py at line 184
	                self = <userplugins.hoster.TusfilesNet.TusfilesNet object at 0x12787b0>
	              thread = <DownloadThread(Thread-2, started daemon)>

Frame process in /share/HDA_DATA/.qpkg/pyload/data/userplugins/internal/SimpleHoster.py at line 306
	              pyfile = PyFile 455: "link".rar@TusfilesNet
	                self = <userplugins.hoster.TusfilesNet.TusfilesNet object at 0x12787b0>

Frame preload in /share/HDA_DATA/.qpkg/pyload/data/userplugins/internal/SimpleHoster.py at line 255
	                self = <userplugins.hoster.TusfilesNet.TusfilesNet object at 0x12787b0>

Frame load in /share/HDA_DATA/.qpkg/pyload/data/userplugins/internal/Plugin.py at line 332
	              decode = True
	             cookies = True
	                 val = False
	                 get = {}
	                 url = u'"link".rar'
	                self = <userplugins.hoster.TusfilesNet.TusfilesNet object at 0x12787b0>
	                 req = <module.network.Browser.Browser object at 0x1282110>
	           multipart = False
	                 key = 'just_header'
	                post = {}
	                 ref = False
	         just_header = False

Frame load in /share/HDA_DATA/.qpkg/pyload/module/network/Browser.py at line 97
	                self = <module.network.Browser.Browser object at 0x1282110>
	                args = (u'"link".rar',
 {},
 {},
 False,
 True,
 False,
 False,
 True)
	              kwargs = {}

Frame load in /share/HDA_DATA/.qpkg/pyload/module/network/HTTPRequest.py at line 202
	             cookies = True
	                 get = {}
	                 url = u'"link".rar'
	                self = <module.network.HTTPRequest.HTTPRequest instance at 0x12acc88>
	              decode = True
	             referer = False
	           multipart = False
	                post = {}
	         just_header = False


PLUGIN OBJECT DUMP: 

	CAPTCHA_BLOCK_PATTERN = '>Enter code.*?<div.*?>(.+?)</div>'
	     CAPTCHA_PATTERN = '(https?://[^"\\\']+?/captchas?/[^"\\\']+)'
	          CHECK_FILE = True
	       CHECK_TRAFFIC = False
	             COOKIES = True
	         DIRECT_LINK = None
	         DISPOSITION = True
	       ERROR_PATTERN = '(?:class=["\\\']err["\\\'].*?>|<[Cc]enter><b>|>Error</td>|>\\(ERROR:)(?:\\s*<.+?>\\s*)*(.+?)(?:["\\\']|<|\\))'
	         FILE_ERRORS = [('Html error',
  '\\A(?:\\s*<.+>)?((?:[\\w\\s]*(?:[Ee]rror|ERROR)\\s*\\:?)?\\s*\\d{3})(?:\\Z|\\s+)'),
 ('Request error', '([Aa]n error occured while processing your request)'),
 ('Html file', '\\A\\s*<!DOCTYPE html')]
	     FORM_INPUTS_MAP = None
	        FORM_PATTERN = None
	  HAPPY_HOUR_PATTERN = '>[Hh]appy hour'
	       HOSTER_DOMAIN = 'tusfiles.net'
	        INFO_PATTERN = '\\](?P<N>.+) - (?P<S>[\\d.,]+) (?P<U>[\\w^_]+)\\['
	        LEECH_HOSTER = True
	   LINK_FREE_PATTERN = '(?:file: "(.+?)"|(https?://(?:www\\.)?([^/]*?tusfiles\\.net|\\d+\\.\\d+\\.\\d+\\.\\d+)(\\:\\d+)?(/d/|(/files)?/\\d+/\\w+/).+?)["\\\'<])'
	  LINK_LEECH_PATTERN = '<h2>Download Link</h2>\\s*<textarea[^>]*>([^<]+)'
	        LINK_PATTERN = '(?:file: "(.+?)"|(https?://(?:www\\.)?([^/]*?tusfiles\\.net|\\d+\\.\\d+\\.\\d+\\.\\d+)(\\:\\d+)?(/d/|(/files)?/\\d+/\\w+/).+?)["\\\'<])'
	LINK_PREMIUM_PATTERN = '(?:file: "(.+?)"|(https?://(?:www\\.)?([^/]*?tusfiles\\.net|\\d+\\.\\d+\\.\\d+\\.\\d+)(\\:\\d+)?(/d/|(/files)?/\\d+/\\w+/).+?)["\\\'<])'
	       LOGIN_ACCOUNT = False
	       LOGIN_PREMIUM = False
	        NAME_PATTERN = '(Filename[ ]*:[ ]*</b>(</td><td nowrap>)?|name="fname"[ ]+value="|<[\\w^_]+ class="(file)?name">)\\s*(?P<N>.+?)(\\s*<|")'
	   NAME_REPLACEMENTS = [('&#?\\w+;', <function fixup at 0xbd71f0>)]
	     OFFLINE_PATTERN = '>\\s*\\w+ (Not Found|file (was|has been) removed|no longer available)'
	PREMIUM_ONLY_PATTERN = '>This file is available for Premium Users only'
	   RECAPTCHA_PATTERN = None
	        SIZE_PATTERN = '(Size[ ]*:[ ]*</b>(</td><td>)?|File:.*>|</font>\\s*\\(|<[\\w^_]+ class="size">)\\s*(?P<S>[\\d.,]+)\\s*(?P<U>[\\w^_]+)'
	   SIZE_REPLACEMENTS = []
	  SOLVEMEDIA_PATTERN = None
	TEMP_OFFLINE_PATTERN = '>\\s*\\w+ server (is in )?(maintenance|maintainance)'
	       TEXT_ENCODING = True
	    URL_REPLACEMENTS = []
	        WAIT_PATTERN = '<span id="countdown_str".*>(\\d+)</span>|id="countdown" value=".*?(\\d+).*?"'
	             account = False
	             captcha = <userplugins.internal.Captcha.Captcha object at 0x1278290>
	         chunk_limit = -1
	           direct_dl = False
	                html = ''
	                info = {'folder': u'1.rar',
 'name': u'1.rar',
 'pattern': {'N': u'1.rar',
             'S': u'101.0',
             'U': u'MB'},
 'size': 105906176.0,
 'status': 2,
 'url': u'"link".rar'}
	                  js = <module.common.JsEngine.JsEngine instance at 0xf55a08>
	          last_check = None
	       last_download = ''
	            leech_dl = False
	             limitDL = 0
	                link = ''
	             multiDL = True
	             premium = False
	              pyfile = PyFile 455: "link".rar@TusfilesNet
	              pyload = <__main__.Core object at 0xc903b0>
	                 req = <module.network.Browser.Browser object at 0x1282110>
	     resume_download = True
	             retries = {}
	          retry_free = False
	              thread = <DownloadThread(Thread-2, started daemon)>
	                user = None
	          wait_until = 0
	             waiting = False
	       wantReconnect = False

PYFILE OBJECT DUMP: 
	               _size = 105906176
	               abort = True
	              active = False
	               error = ''
	                  id = 455
	                lock = <_RLock(None, 0)>
	                   m = <module.database.FileDatabase.FileHandler instance at 0xcf2f58>
	         maxprogress = 100
	                name = u'"link".rar'
	               order = 9
	           packageid = 27
	              plugin = <userplugins.hoster.TusfilesNet.TusfilesNet object at 0x12787b0>
	         pluginclass = <class 'userplugins.hoster.TusfilesNet.TusfilesNet'>
	        pluginmodule = <module 'userplugins.hoster.TusfilesNet' from '/share/HDA_DATA/.qpkg/pyload/data/userplugins/hoster/TusfilesNet.pyc'>
	          pluginname = u'TusfilesNet'
	            progress = 0
	         reconnected = False
	                size = 105906176
	              status = 8
	          statusname = None
	                 url = u'"link".rar'
	           waitUntil = 0
 ```

**Comment by Dybala93** [comment]
After fixing update manager and COOKIES problem for now all seems works.
I wait tomorrow for close.
Thanks!
